src/test/java/bio/terra/service/snapshotbuilder/utils/HierarchyQueryBuilderTest.java

```diff
@@ -28,10 +28,11 @@ class HierarchyQueryBuilderTest {
       FROM concept_relationship AS c
                JOIN concept AS c0 ON c0.concept_id = c.concept_id_2
                JOIN concept AS c4 ON c4.concept_id = c.concept_id_1
-               LEFT JOIN condition_occurrence AS c1 ON c1.condition_concept_id = c.concept_id_2
-      WHERE (c.concept_id_1 IN (SELECT c5.ancestor_concept_id
-           FROM concept_ancestor AS c5
-           WHERE (c5.descendant_concept_id = 1 AND c5.ancestor_concept_id != 1)) AND
+               JOIN concept_ancestor AS c5 ON c5.ancestor_concept_id = c.concept_id_2
+               LEFT JOIN condition_occurrence AS c1 ON c1.condition_concept_id = c5.descendant_concept_id
+      WHERE (c.concept_id_1 IN (SELECT c6.ancestor_concept_id
+           FROM concept_ancestor AS c6
+           WHERE (c6.descendant_concept_id = 1 AND c6.ancestor_concept_id != 1)) AND
```

Comment:
I think I understand the changes in L31 & L32 but I don't understand why this where clause changed. Can you point to the code changes in the HierarchyQueryBuilder that are causing this to change?

Reply:
Oh, sorry, I was for some reason thinking the nested select in L33-35 was new but it's not. I see it now. This makes sense!
I guess reading raw SQL changes can be a little confusing!